js/languagePickerDrawerView.js

```diff
@@ -29,7 +29,8 @@ define([
       var data = this.model.getLanguageDetails(newLanguage);
 
       var promptObject = {
-        _classes: 'is-ltr',
+        _attributes: { lang: newLanguage },
+        _classes: 'is-ltr is-lang-' + newLanguage,
```

Comment:
```suggestion
        _classes: `is-ltr is-lang-${newLanguage}`,
```
Sorry just realised that because this is dependent on an update to the FW it'll a) need a FW dependency bump (you'll have to wait until the updated FW version is released for that, though) and b) you can use a "template string" to make it look slightly neater

Reply:
> to make it look slightly neater

arguably, anyway... your call as to which style you prefer!

Reply:
@NayanKhedkar if you could bump the FW dependency to ">=5.6" we can get this one moving again

Reply:
sure